core/src/main/java/io/questdb/std/Files.java

```diff
@@ -103,19 +103,15 @@ public static long ceilPageSize(long size) {
     }
 
     public static int close(int fd) {
-        assert auditClose(fd);
-        int res = close0(fd);
-        if (res == 0) {
-            OPEN_FILE_COUNT.decrementAndGet();
-        }
-        return res;
-    }
-
-    // If fd is negative, no action occurs.
-    public static int closeChecked(int fd) {
-        if (fd >= 0) {
-            return close(fd);
+        if (fd > 0) {
```

Comment:
0 - is a valid descriptor, and should not be ignored

Reply:
I also checked that and 0 always stands for stdin unless you explicitly do `close(0)` which is definitely not what we want to do. Still, I agree that a `if (fd > -1)` would be less questionable.

Reply:
it should be `> -1` or `> 2` then

Reply:
sure

Reply:
done in https://github.com/questdb/questdb/pull/2938/commits/a4f474d1cbcd29e57b5011bf2af1ced9db2b0dc3